src/witan/models/dem/ccm/mort/mortality.clj

```diff
@@ -55,75 +55,106 @@
+                                                                                                :death-rate
+                                                                                                :death-rate
+                                                                                                start-yr-avg-mort
+                                                                                                end-yr-avg-mort)]
+                                                               {:initial-projected-mortality-rates
+                                                                (cf/apply-national-trend projected-rates-jumpoffyr
+                                                                                         future-mortality-trend-assumption
+                                                                                         first-proj-yr
+                                                                                         last-proj-yr
+                                                                                         mort-scenario
+                                                                                         :death-rate)})
+                                 :applynationaltrend-trend (let [projected-rates-jumpoffyr
+                                                                 (cf/jumpoffyr-method-trend historic-asmr
+                                                                                            :death-rate
+                                                                                            :death-rate
+                                                                                            start-yr-avg-mort
+                                                                                            end-yr-avg-mort)]
+                                                             {:initial-projected-mortality-rates
+                                                              (cf/apply-national-trend projected-rates-jumpoffyr
+                                                                                       future-mortality-trend-assumption
+                                                                                       first-proj-yr
+                                                                                       last-proj-yr
+                                                                                       mort-scenario
+                                                                                       :death-rate)}))]
+    asmr-projection-method))
 
-(defworkflowfn project-deaths-from-fixed-rates
-  "Takes a dataset with population at risk from the current year of the projection
-  loop and another dataset with fixed death rates for the population. Returns a
-  dataset with a column of deaths, which are the product of popn at risk & death rates"
-  {:witan/name :ccm-mort/project-deaths-fixed-rates
+(defworkflowfn project-asmr-1-0-0
```

Comment:
@sunnytownsend At the moment project-asmr has a default method, which we have defined as average-fixed. I remember you said we wanted to change the default method probably to trend-fixed instead, as that (might) make more sense as a default. This may be cause for a new version, but I would strongly suggest making this change on a new branch, to keep things clear and it would require fixing/changing tests I believe.

What is important is that this step is always going to be projecting the ASMR, but potentially using different methods along the way. If we keep the original naming convention we may as well return to having four separate workflowfns (a possibility)